code/components/extra-natives-five/src/InterfaceExtraNatives.cpp

```diff
@@ -43,13 +152,42 @@ static char LoadZoomMapDataMeta()
 	return success;
 }
 
+static void PatchResetHudComponentValues()
+{
+	static constexpr uint64_t nativeHash = 0x450930E616475D0D;
+
+	auto handler = fx::ScriptEngine::GetNativeHandler(nativeHash);
+
+	if (!handler)
+	{
+		trace("Couldn't find 0x%08x handler to hook!\n", nativeHash);
+		return;
+	}
+
+	fx::ScriptEngine::RegisterNativeHandler(nativeHash, [=](fx::ScriptContext& ctx)
+	{
+		auto index = ctx.GetArgument<uint32_t>(0);
+
+		if (auto& it = hudComponentOldEntries.find(index); it != hudComponentOldEntries.end())
+		{
+			auto component = &hudComponentsArray[index];
+			component->alignX[0] = it->second.alignX[0];
+			component->alignY[0] = it->second.alignY[0];
+			component->sizeX = it->second.sizeX;
+			component->sizeY = it->second.sizeY;
+		}
+
+		return (*handler)(ctx);
+	});
+}
+
 static HookFunction initFunction([]()
 {
 	{
 		auto location = hook::get_pattern("33 C0 0F 57 C0 ? 0D", 0);
 
-		expandedRadar = hook::get_address<uint64_t*>((char*)location + 7);
-		revealFullMap = hook::get_address<uint64_t*>((char*)location + 37);
+		expandedRadar = hook::get_address<bool*>((char*)location + 7);
```

Comment:
For easier change tracking, can this type fix be moved to a _separate_ commit?

(it can be in the same PR still, I think)